src/components/Button/index.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./Button.css";
+
+function Button({ text }) {
```

Comment:
@AbdurrahmanSogbesan  there are other props to add here. onChange, disabled to name a few